airbyte-integrations/connectors/source-fauna/examples/config_sample.json

```diff
@@ -0,0 +1,16 @@
+{
+  "secret": "SECRET",
+  "domain": "db.fauna.com",
```

Comment:
I tried this with a test database that I created on https://dashboard.fauna.com/, but it did not work. The error I get is `Failed to connect to database: Unauthorized`. I am using the secret key from https://dashboard.fauna.com.

Given that Fauna has a cloud based service, it would make sense to have an easy to follow example for setting up a connection to cloud hosted Fauna database.

Reply:
Did you set the `domain` field correctly? It needs to be set based on the region you chose.

See the region group docs: https://docs.fauna.com/fauna/current/learn/understanding/region_groups#how-to-use-region-groups

Reply:
Thanks for the pointer! I did not set the domain correctly as it turns out. I couldn't find those details on my Fauna cloud dashboard. I will retest

Reply:
Yeah, we're going to fix that Soon™ :)

I'm glad it works! Let me know how your test goes.